### PR TITLE
CMakeLists.txt: Check for LLVM 7 or above.

### PR DESCRIPTION
**Pull request by TurboGit**
Just to avoid having to change the file for every new version!

### CODE REVIEW

**Comment by kmilos** [comment]
Nice. Might want to do something similar for libheif as well.

**Comment by TurboGit** [comment]
> Nice. Might want to do something similar for libheif as well.

Thanks, I take this as a big win for me as I really just know next to nothing about CMake :)

File: CMakeLists.txt [hunk 1]
```diff
@@ -313,17 +313,11 @@ endif(VALIDATE_APPDATA_FILE)
...
-      if(LLVM_FOUND)
-        break()
-      endif()
-    endforeach()
-  endmacro()
+  find_package(LLVM CONFIG)
 
-  # 3.9 is the first version with which this works.
-  find_llvm("15;14;13;12;11;10;9;8;7;6.0;5.0;4.0;3.9")
+  if(${LLVM_PACKAGE_VERSION} LESS 7)
```
Comment (parafin) [inline CMakeLists.txt:318]: should probably be VERSION_LESS instead of just LESS

Comment (kmilos) [reply CMakeLists.txt]: Should be simply `LLVM_VERSION`

Comment (TurboGit) [reply CMakeLists.txt]: Testing that! Thanks.

**Comment by TurboGit** [comment]
All green now! Thanks for the help.

**Pull request merged**